src/game/Tactical/Soldier_Profile.cc

```diff
@@ -120,8 +120,8 @@ static void StartSomeMercsOnAssignment(void);
 
 void LoadMercProfiles()
 {
-	{ AutoSGPFile f(GCM->openGameResForReading(BINARYDATADIR "/prof.dat"));
-		LoadRawMercProfiles(f, NUM_PROFILES, gMercProfiles, getDataFilesEncodingCorrector());
```

Comment:
Nice, looks like this was the only user of `getDataFilesEncodingCorrector` which means not only this function but the entire `IEncodingCorrector` interface can go. You don't have to do that in this PR but if it's not one of your planned followup PRs please let me know so I don't forget it.

Reply:
No, It's not in my plans